frontend/.prettierrc.json

```diff
@@ -1,6 +1,4 @@
 {
-  "trailingComma": "es5",
-  "semi": false,
-  "singleQuote": true,
-  "jsxSingleQuote": true
```

Comment:
Any reasoning for switching from single quotes?

Reply:
Good question, I usually like to leave this as default as possible. I would usually only set trailingComma to `all` and nothing else. Decided to keep semi `false`, as it seems pretty widespread in the solution.